### PR TITLE
Feature/resize items

### PR DESCRIPTION
**Pull request by vasdee**
fair few merge conflicts in this, I think i managed to get them all across correctly.

### CODE REVIEW

File: src/timeline.js [hunk 1]
```diff
@@ -9,11 +9,19 @@ import interact from 'interactjs';
...
+  getNearestRowHeight,
+  getMaxOverlappingItems,
+  getDurationFromPixels
+} from 'utils/itemUtils';
 import {groupRenderer} from 'utils/groupUtils';
-
 import Timebar from 'components/timebar';
+
 import './style.css';
+import {isThisSecond} from 'date-fns';
```
Comment (lilfolr) [inline src/timeline.js]: What's this for? Is it a moment alternative thing?

File: src/timeline.js [hunk 2]
```diff
@@ -206,7 +252,7 @@ export default class Timeline extends Component {
       return ({index}) => {
         if (columnCount == 1) return width;
         if (columnCount == 2) {
-          let groupWidth = 100;
+          let groupWidth = 150;
```
Comment (lilfolr) [inline src/timeline.js]: CHANGE TO 100

File: src/timeline.js [hunk 3]
```diff
@@ -80,18 +89,14 @@ export default class Timeline extends Component {
...
+      })
+      .on('dragstart', e => {
         e.target.style['z-index'] = 2;
-        //TODO: This should use state reducer
-        //TODO: Should be able to optimize the lookup below
-        const index = e.target.getAttribute('item-index');
-        const rowNo = this.itemRowMap[index];
-        const itemIndex = _.findIndex(this.rowItemMap[rowNo], i => i.key == index);
-        const item = this.rowItemMap[rowNo][itemIndex];
-        this.setSelection(item.start, item.end);
```
Comment (lilfolr) [inline src/timeline.js]: Did you delete all my dragstart logic :(

File: src/timeline.js (hunk 1 above)
Comment (vasdee) [reply src/timeline.js]: i thought you added it in, i'll remove

File: src/timeline.js (hunk 2 above)
Comment (lilfolr) [reply src/timeline.js]: You dont need to change this anymore - ive refactored to be prop driven

**Pull request merged**